src/db/service/user.service.ts

```diff
@@ -155,7 +156,7 @@ export default class UserService {
 			return messages.internalError(error.message);
 		}
 	}
-
+	/* istanbul ignore next */
 	async validatePassword(body: any) {
```

Comment:
Can we have a test for this function `validatePassword(..)`?

Reply:
we can provide a mocked body here

Reply:
my bad